planning/behavior_path_planner/src/utils/avoidance/utils.cpp

```diff
@@ -1677,4 +1693,145 @@ std::pair<PredictedObjects, PredictedObjects> separateObjectsByPath(
 
   return std::make_pair(target_objects, other_objects);
 }
+
+DrivableLanes generateExpandDrivableLanes(
+  const lanelet::ConstLanelet & lanelet, const std::shared_ptr<const PlannerData> & planner_data,
+  const std::shared_ptr<AvoidanceParameters> & parameters)
+{
+  const auto & route_handler = planner_data->route_handler;
+
+  DrivableLanes current_drivable_lanes;
+  current_drivable_lanes.left_lane = lanelet;
+  current_drivable_lanes.right_lane = lanelet;
+
+  if (!parameters->use_adjacent_lane) {
+    return current_drivable_lanes;
+  }
+
+  const auto has_same_lane =
+    [](const lanelet::ConstLanelets lanes, const lanelet::ConstLanelet & lane) {
+      if (lanes.empty()) return false;
+      const auto has_same = [&](const auto & ll) { return ll.id() == lane.id(); };
+      return std::find_if(lanes.begin(), lanes.end(), has_same) != lanes.end();
```

Comment:
using `std::any_of` would give more accurate nuance.

Reply:
@zulfaqar-azmi-t4 Thanks for your advice! I fixed in https://github.com/autowarefoundation/autoware.universe/pull/5018/commits/e9fe60525d3e6510ba7957bf91b53280da726f47.

Reply:
thank you Ota-san!